README.md

```diff
@@ -1,7 +1,7 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg-bright.png">
-  <source media="(prefers-color-scheme: light)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
-  <img alt="Nextflow Logo" src="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow-logo-bg-dark.png">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow-logo-bg-light.png">
+  <img alt="Nextflow Logo" src="https://github.com/nextflow-io/trademark/blob/master/nextflow-logo-bg-light.png">
```

Comment:
Please update to use only assets in this repo (add in new logo images if needed).

x-ref https://github.com/nextflow-io/nextflow/issues/4395#issuecomment-1760339290